gyp/pylib/gyp/generator/make.py

```diff
@@ -103,6 +103,8 @@ def CalculateVariables(default_variables, params):
             default_variables.setdefault("SHARED_LIB_SUFFIX", ".x")
         else:
             default_variables.setdefault("SHARED_LIB_SUFFIX", ".so")
+        if flavor == "zos":
+            COMPILABLE_EXTENSIONS.update({".pli": "pli"})
```

Comment:
Would it make more sense for this to be grouped together with the `elif flavor == "zos":` on line 102-103?

Reply:
Good catch!